packages/connect-extension-protocol/src/index.ts

```diff
@@ -51,13 +51,13 @@ interface ToExtensionHeader {
 interface ToExtensionAddChain {
   type: "add-chain"
   chainId: string
-  payload: { chainSpec: string; parachainSpec?: string }
+  payload: { chainSpec: string; potentialRelayChainIds: Array<string> }
 }
 
 interface ToExtensionAddWellKnownChain {
   type: "add-well-known-chain"
   chainId: string
-  payload: { name: string; parachainSpec?: string }
+  payload: string
```

Comment:
```suggestion
  chainSpec: string
```

Reply:
nope, that's not correct... It's the name of the "well-known-chain" what's passed in the payload... If you want I can change the payload to `payload: { name: string }`.

There is a reason for putting everything under `payload`, which is that it makes basic message assertions easier to deal with.